src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString.split(';');
```

Comment:
The variable name 'cssProperties' could be more descriptive. Consider renaming it to reflect that it contains an array of style declarations, such as 'styleDeclarations' or 'cssDeclarations'.